security-command-center/snippets/src/main/java/vtwo/iam/GetIamPolicies.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo.iam;
+
+// [START securitycenter_get_iam_policies_v2]
+
+import com.google.cloud.securitycenter.v2.SecurityCenterClient;
+import com.google.cloud.securitycenter.v2.SourceName;
+import com.google.iam.v1.GetIamPolicyRequest;
+import com.google.iam.v1.GetPolicyOptions;
+import com.google.iam.v1.Policy;
+import java.io.IOException;
+
+public class GetIamPolicies {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(Developer): Replace the sample resource name
+    // organizationId: Google Cloud Organization id.
+    String organizationId = "{google-cloud-organization-id}";
+
+    // The source id corresponding to the finding.
+    String sourceId = "{source-id}";
+
+    getIamPolicySource(organizationId, sourceId);
+  }
+
+  // Demonstrates how to retrieve IAM policies for a source
+  public static void getIamPolicySource(String organizationId, String sourceId) {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      // Start setting up a request to get IAM policy for a source.
+      SourceName sourceName = SourceName.ofOrganizationSourceName(organizationId, sourceId);
+
+      GetIamPolicyRequest request = GetIamPolicyRequest.newBuilder()
+          .setResource(sourceName.toString())
+          .setOptions(GetPolicyOptions.newBuilder().build())
+          .build();
+
+      // Call the API.
+      Policy response = client.getIamPolicy(request);
+      System.out.println("Policy: " + response);
+    } catch (IOException e) {
+      System.out.println("get iam policy failed! \n Exception: " + e);
+    }
```

Comment:
nit: please replace print outs to the status response. Also either narrow the exception catch up or propagate it up instead of catching everything related to IO

Reply:
Your suggestion makes sense, but looking at other code samples for v2, it seems to me that it would deviate from the standard.
https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/security-command-center/snippets/src/main/java/vtwo

Reply:
There is no standard about it. This implementation was used to reduce overhead on test creation when code snippets without tests were originally migrated from other repos.
We are trying to reduce test flakiness by requiring more robust test validations in new tests.

Reply:
done